static/app/views/replays/replays.tsx

```diff
@@ -15,26 +15,23 @@ import {IconArrow, IconCalendar} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import {PageContent, PageHeader} from 'sentry/styles/organization';
 import space from 'sentry/styles/space';
-import {NewQuery, Organization, PageFilters} from 'sentry/types';
+import {NewQuery, PageFilters} from 'sentry/types';
 import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
 import EventView from 'sentry/utils/discover/eventView';
 import {generateEventSlug} from 'sentry/utils/discover/urls';
 import {useLocation} from 'sentry/utils/useLocation';
 import useOrganization from 'sentry/utils/useOrganization';
 import useProjects from 'sentry/utils/useProjects';
-import withOrganization from 'sentry/utils/withOrganization';
 import withPageFilters from 'sentry/utils/withPageFilters';
 import AsyncView from 'sentry/views/asyncView';
 
 import ReplaysFilters from './filters';
 import {Replay} from './types';
 
-type Props = AsyncView['props'] &
-  WithRouterProps<{orgId: string}> & {
-    organization: Organization;
-    selection: PageFilters;
-    statsPeriod?: string | undefined; // revisit i'm sure i'm doing statsperiod wrong
-  };
+type Props = AsyncView['props'] & {
+  selection: PageFilters;
+  statsPeriod?: string | undefined; // revisit i'm sure i'm doing statsperiod wrong
```

Comment:
Is this needed?

Reply:
I don't think so, it was there before I started working in this codebase. I'll remove it if it's not needed.